### PR TITLE
Add support for MSVC

### LINKED ISSUE #59: Add support for MSVC
See [here](https://cgit.freedesktop.org/pkg-config/tree/README.win32). In theory this should work by just passing `--msvc-syntax` to pkg-config.

### PR DESCRIPTION
**Pull request by ferjm**
Fixes #59

### CODE REVIEW

**Comment by ferjm** [comment]
I tested this building [glib-sys](https://github.com/gtk-rs/sys/tree/master/glib-sys) and it seems to work properly.

File: src/lib.rs [hunk 1]
```diff
@@ -410,6 +393,10 @@ impl Config {
         if self.is_static(name) {
             cmd.arg("--static");
         }
+        let target = env::var("TARGET").unwrap_or_else(|_| String::new());
+        if target.contains("msvc") {
```
Comment (sdroege) [inline src/lib.rs]: Why not
```rust
if let Some(target) = env::var("TARGET") {
    if target.contains("msvc") {
        cmd.arg("--msvc-syntax");
    }
}
```
or something with match and guards for the `contains`?
One level of indentation more, but doesn't have the empty string

**Comment by sdroege** [comment]
Looks good to me and should work, at least better than what we have currently. As @nirbheek said in #59, `--msvc-syntax` had some issues in the past in some corner cases but we can worry about that when it happens.

**Comment by sdroege** [comment]
@alexcrichton Thanks! Can you prepare a new release with that change?

**Comment by alexcrichton** [comment]
Sure thing, done!

**Pull request merged**